### DISCUSSION TITLE
Button icon position right

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#42555] [Ideas]
Would be great having the icon also on the right side.

https://mantine.dev/core/button/

**Comment by h-bessa** [comment]
Up. Any new about this enhancement ?

**Reply by anderybis** [reply]
`<Button>Сортировка{<SortAscendingOutlined />}</Button>`

**Reply by h-bessa** [reply]
thank @anderybis

**Comment by afc163** [comment]
- #47791
